ui/web-v2/apps/admin/src/pages/index.tsx

```diff
@@ -51,6 +54,25 @@ import { SegmentIndexPage } from './segment';
 import { SettingsIndexPage } from './settings';
 
 export const App: FC = memo(() => {
+  const location = useLocation();
+  const [initialized, setInitialized] = useState(false);
+
+  useEffect(() => {
+    if (
+      !window.location.href.includes('localhost') &&
```

Comment:
@bimalgrg519, please add this `RELEASE_CHANNEL` const in the config file.
And verify if it is `prod` before initializing the GA.

E.g.

```js
export const RELEASE_CHANNEL = process.env.RELEASE_CHANNEL;

if (
  RELEASE_CHANNEL === 'prod' &&
  !window.location.href.includes('localhost') &&
  GOOGLE_ANALYTICS_ID.trim().length > 0
)
```

Please update the other places using `process.env.RELEASE_CHANNEL` when you add the const.

Reply:
@bimalgrg519, nevermind. Leave it as it is.